scripts/remote-tmux-live-fuzz.sh

```
@@ -0,0 +1,268 @@
+    tries=$((tries + 1))
+    sleep 2
+  done
+  if [ "$tries" -ge 10 ]; then
+    note_fail "$iter" "windows never settled after 20s: $(printf '%s' "$settled_json" | tr -d '\n' | cut -c1-300)"
+  elif printf '%s' "$settled_json" | grep -q 'rendered='; then
+    note_fail "$iter" "settled with pane mismatches:"
+    printf '%s' "$settled_json" | grep -o '"%[0-9]* rendered=[^"]*"' | head -4 | sed 's/^/  /'
+  fi
+  # Oracle 2: rendered text equals tmux truth for every pane.
+  local ws
+  ws=$("$CLI" list-workspaces 2>/dev/null | awk '/fuzz|\[selected\]/{print $2; exit}')
```

Shellcheck (0.11.0)

[warning] ws appears unused. Verify use (or export if used externally).

(SC2034)